Header header
int16 self_id
clt_random_datasets/RobotObservation[] observations
